### PR TITLE
Changed first example function in Errors and Exceptions to execute a …

### PR DESCRIPTION
**Pull request by aahowel3**
…practical command rather than a list of ice creams. In student's experience functions are designed to do something. New example function also incorporates if/else statements learned in previous lesson.

### CODE REVIEW

File: _episodes/07-errors.md [hunk 1]
```diff
@@ -70,16 +66,16 @@ You can determine the number of levels by looking for the number of arrows on th
 In this case:
 
 1.  The first shows code from the cell above,
-    with an arrow pointing to Line 8 (which is `favorite_ice_cream()`).
+    with an arrow pointing to Line 9 (which is `lucky_dice(3)`).
```
Comment (maxim-belkin) [inline _episodes/07-errors.md]: Line 1 ?

Comment (aahowel3) [reply _episodes/07-errors.md]: 1 as in the first level of the traceback.

Comment (maxim-belkin) [reply _episodes/07-errors.md]: my bad! `Line 9` is correct

**Comment by maxim-belkin** [comment]
Thanks for the PR, Abigail. I'm not yet sure what I think about it, but the content-wise it looks good to me. If you could provide a better justification as to why this part should be changed - that would be very much appreciated.

File: _episodes/07-errors.md [hunk 2]
```diff
@@ -34,32 +34,28 @@ Let's examine one:
...
-        "strawberry"
-    ]
-    print(ice_creams[3])
-
-favorite_ice_cream()
+def lucky_dice(number):
+    outcomes=['win','lose']
+    if number >= 6:
+        print(outcomes[1])
+    if number < 6:
```
Comment (maxim-belkin) [inline _episodes/07-errors.md]: else?

Comment (annefou) [reply _episodes/07-errors.md]: Yes definitely "else". I am not sure this new example is clearer. I agree the initial example can be found a bit unrealistic but the main advantage is that it is fairly straightforward. I wonder if we would divert attendees by adding complexity while our main objective is to demonstrate how python tackles errors.

File: _episodes/07-errors.md [hunk 3]
```diff
@@ -34,32 +34,28 @@ Let's examine one:
...
+        print(outcomes[2])
+lucky_dice(3)
```
Comment (maxim-belkin) [inline _episodes/07-errors.md]: what if we change our example to:

```python
def select_favorite(items):
    print(items[3])

ice_cream = ["chocolate",  "vanilla", "strawberry"]

select_favorite(ice_cream)
```

Would that be any better?

**Pull request closed without merging**